src/main/java/com/robosh/ejournal/entity/School.java

```diff
@@ -0,0 +1,33 @@
+package com.robosh.ejournal.entity;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
```

Comment:
why we have here two empty lines? In the next file we have only one empty string between different package imports. Please use ctrl+alt+L in all files